cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/store/DefaultMetadataStore.java

```diff
@@ -465,13 +466,21 @@ public void apply(MetadataDataset input) throws Exception {
 
   @Override
   public Set<MetadataRecord> getSnapshotBeforeTime(MetadataScope scope, final Set<Id.NamespacedId> entityIds,
-                                                    final long timeMillis) {
-    return execute(new TransactionExecutor.Function<MetadataDataset, Set<MetadataRecord>>() {
+                                                   final long timeMillis) {
+    Set<MetadataHistoryEntry> metadataHistoryEntries = execute(new TransactionExecutor.Function<MetadataDataset,
+      Set<MetadataHistoryEntry>>() {
```

Comment:
nit: the indentation seems a little weird here. It seems like the lines below should be indented one more level. Will it be better if you move the entire ``execute...`` to the next line? .